anchor/functions/search.php

```diff
@@ -155,14 +151,24 @@ function search_form_input($extra = '')
     return '<input name="term" type="text" ' . $extra . ' value="' . search_term() . '">';
 }
 
+/**
+ * Get the search item object from the registry.
+ *
+ * @return Object
+ */
+function search_item()
+{
+    return Registry::get('search_item');
+}
+
 /**
  * Retrieves the search item type (post/page/other)
  *
  * @return string
  */
 function search_item_type()
 {
-    $item_class = strtolower(get_class(Registry::get('search_item')));
+    $item_class = strtolower(get_class(search_item()));
```

Comment:
Don't we actually have an `itemtype` or `pagetype` property on these models we could use instead of `get_class`? I think I stumbled across that somewhere.

Reply:
Probably, I guess this is just another way to do it?